### PR TITLE
Extra volumes

### PR DESCRIPTION
**Pull request by mikebreed**
This PR enables the option to provide extra volumes and extra volume mounts to each of the Kobo containers. This could be useful for a number of occasions such as injecting additional CA certificates or config files. This was modeled after similar features on Bitnami charts and other open source helm charts.

### CODE REVIEW

File: templates/kpi/deployment.yaml [hunk 1]
```diff
@@ -36,17 +36,17 @@ spec:
...
-              path: /accounts/login/
-              port: http
-            initialDelaySeconds: 5
-            timeoutSeconds: 5
-          readinessProbe:
-            httpGet:
-              path: /accounts/login/
-              port: http
-            initialDelaySeconds: 5
+{{/*          livenessProbe:*/}}
```
Comment (bufke) [inline templates/kpi/deployment.yaml:39]: Was this a mistake?

Comment (mikebreed) [reply templates/kpi/deployment.yaml:39]: Yes, sorry about that. I'll fix this

Comment (mikebreed) [reply templates/kpi/deployment.yaml]: > Was this a mistake?

This is fixed along with the other erroneous changes

**Comment by bufke** [comment]
Thanks

**Pull request merged**